src/mlpack/tests/size_checks_test.cpp

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file size_checks_test.cpp
+ * @author Bisakh Mondal
+ * 
+ * Test file for Utility size_checks.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <boost/test/unit_test.hpp>
+
+using namespace mlpack;
+using namespace mlpack::util;
+BOOST_AUTO_TEST_SUITE(SizeCheckTest);
```

Comment:
```suggestion
using namespace mlpack::util;

BOOST_AUTO_TEST_SUITE(SizeCheckTest);
```
Just a little spacing might be nice here. :)

Reply:
:+1: